aries-site/src/data/wcag/rangeinput.json

```diff
@@ -0,0 +1,512 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "passed",
```

Comment:
this would be conditional the implementor needs to be sure to pass a `aria-label` for RangeInput